File: xwallpaper/src/main/java/vn/loitp/app/activity/customviews/textview/colortextview/ColorTextViewActivity.java
package vn.loitp.app.activity.customviews.textview.colortextview;

import android.app.Activity;
import android.os.Bundle;

import vn.loitp.app.activity.customviews.textview.colortextview.lib.ColorTextView;
import vn.loitp.app.base.BaseActivity;
import com.loitp.xwallpaper.R;

public class ColorTextViewActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        //can set color in xml or java code
        /*((ColorTextView) findViewById(R.id.ctv_text))
                .findAndSetStrColor("after", "#ff8000")
                .findAndSetStrColor("and", "#008888");*/
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected Activity setActivity() {
        return this;
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_color_textview;
    }

}

File: app/src/main/java/vn/loitp/app/a/customviews/textview/colorTextView/ColorTextViewActivity.kt
package vn.loitp.app.a.customviews.textview.colorTextView

import android.os.Bundle
import com.loitp.annotation.IsFullScreen
import com.loitp.annotation.LogTag
import com.loitp.core.base.BaseFontActivity
import com.loitp.core.utilities.LUIUtil
import kotlinx.android.synthetic.main.activity_text_view_color.*
import vn.loitp.R

@LogTag("ColorTextViewActivity")
@IsFullScreen(false)
class ColorTextViewActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_text_view_color
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupViews()
    }

    private fun setupViews() {
        lActionBar.apply {
            LUIUtil.setSafeOnClickListenerElastic(
                view = this.ivIconLeft,
                runnable = {
                    onBaseBackPressed()
                }
            )
            this.ivIconRight?.setImageResource(R.color.transparent)
            this.tvTitle?.text = ColorTextViewActivity::class.java.simpleName
        }
    }
}
